webpack.renderer.config.js

```diff
@@ -8,16 +8,22 @@ const fs = require(`fs-extra`)
 
 const HtmlWebpackPlugin = require(`html-webpack-plugin`)
 const VueLoaderPlugin = require(`vue-loader/lib/plugin`)
+const BundleAnalyzerPlugin = require(`webpack-bundle-analyzer`)
+  .BundleAnalyzerPlugin
+const CleanWebpackPlugin = require(`clean-webpack-plugin`)
 
 function resolve(dir) {
   return path.join(__dirname, dir)
 }
 
+const buildPath = path.join(__dirname, `app/dist`)
+
+const devPlugins = process.env.CIRCLECI
+  ? []
+  : [new CleanWebpackPlugin([buildPath]), new BundleAnalyzerPlugin()]
+
 const rendererConfig = {
-  devtool:
-    process.env.NODE_ENV === `production`
-      ? `#source-map`
-      : `#inline-source-map`,
+  devtool: process.env.NODE_ENV === `production` ? false : `#inline-source-map`,
```

Comment:
why? we need source-maps, are they produces somewhere else?

Reply:
But we need them to have proper stack traces with Sentry, don't we?

Reply:
Sentry! Thanks for reminding me! Ok I'll put them back as separate file ok?
cheap-source-map should be a bit lighter but provide the stack trace and module references. Should be readable code although not indented